ods_ci/tests/Resources/Page/ODH/ODHDashboard/ODHDataScienceProject/Pipelines.resource

```diff
@@ -39,9 +39,9 @@ Select Data Connection
     [Documentation]    Selects an existing data connection from the dropdown
     ...                in the modal for Pipeline Server creation
     [Arguments]    ${dc_name}
-    Click Element    css:form div span button[aria-label="Options menu"]
-    Wait Until Page Contains Element    xpath://button[text()="${dc_name}"]
-    Click Element    xpath://button[text()="${dc_name}"]
+    Click Element    xpath://div[@class="pf-v5-c-form__group"][.//label//*[text()="Access key"]]//div[@data-ouia-component-type="PF5/Dropdown"]
```

Comment:
using `-v5` will be proner to breaks since it seems to refer to a patternfly version. I'd rather use something more flexible

Reply:
I'm thinking what to do instead. I'm not seeing a robust targeted xpath to use instead. Having to do search-and-replace for -v5 when upgrading is silly, for sure.

Reply:
we could try with some "contains" check. For example, in the line https://github.com/red-hat-data-services/ods-ci/pull/1113/files#diff-82e844de82ae4101df6d15b896b02ea96f801b4a27c3a102c4e120da077518d1R40 

`Choose File //div[contains(@class="-file-upload")]//input[@type="file"]`

Reply:
I'm not sure how big a win this is, but yeah, I'll be using that, np.

Reply:
I can't tell for sure neither, the DOM could change in many ways under our feet, but we could try reducing the point of failure where we see a little chance.